pkg/controller/master/upgrade/plan_controller_test.go

```diff
@@ -117,6 +117,11 @@ func TestPlanHandler_OnChanged(t *testing.T) {
 			sanitizeStatus(&actual.plan.Status)
 		}
 
+		if tc.expected.upgrade != nil {
+			emptyConditionsTime(tc.expected.upgrade.Status.Conditions)
```

Comment:
seem has same logic in https://github.com/harvester/harvester/blob/a242948beeac83eab142bb5ee073aa53f8a1f2dc/pkg/controller/master/upgrade/plan_controller_test.go#L109